tsconfig.json

```diff
@@ -24,15 +24,27 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
+    "types": [
+      "@emotion/core", 
+      "node",
+      "jest",
+      "@testing-library/jest-dom"
+    ],
     "baseUrl": "src",
     "noFallthroughCasesInSwitch": true,
   },
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "vite.config.ts"
   ],
   "include": [
     "src",
     "test_utils/index.tsx",
     "test_utils/custom-queries.ts"
```

Comment:
Not a part of this PR but you might also want to change this to  `test_utils/custom-queries.js`

Reply:
Great catch! updated the file type.